packages/instrumentation-browser-navigation/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@opentelemetry/instrumentation-browser-navigation",
+  "version": "0.39.0",
+  "description": "OpenTelemetry instrumentation for browser navigation events (page load and same-document navigations)",
+  "main": "build/src/index.js",
+  "module": "build/esm/index.js",
+  "esnext": "build/esnext/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "prewatch": "npm run version:update",
+    "version:update": "node ../../scripts/version-update.js",
+    "compile": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "prepublishOnly": "npm run compile",
+    "tdd": "wtr --watch",
+    "test:browser": "karma start --single-run",
+    "watch": "tsc --build -watch tsconfig.json tsconfig.esm.json tsconfig.esnext.json"
+  },
+  "keywords": [
+    "opentelemetry",
+    "browser",
+    "navigation",
+    "web",
+    "logs",
+    "plugin"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/esm/**/*.js",
+    "build/esm/**/*.map",
+    "build/esm/**/*.d.ts",
+    "build/esnext/**/*.js",
+    "build/esnext/**/*.map",
+    "build/esnext/**/*.d.ts",
+    "build/src/**/*.js",
+    "build/src/**/*.map",
+    "build/src/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.3.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.22.17",
+    "@jsdevtools/coverage-istanbul-loader": "3.0.5",
+    "@opentelemetry/api": "^1.3.0",
+    "@opentelemetry/api-logs": "^0.203.0",
+    "@opentelemetry/sdk-logs": "^0.203.0",
+    "@types/node": "18.18.14",
+    "@types/sinon": "17.0.4",
+    "karma": "6.4.4",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-coverage": "2.2.1",
+    "karma-jquery": "0.2.4",
+    "karma-mocha": "2.0.1",
+    "karma-spec-reporter": "0.0.36",
+    "karma-webpack": "5.0.1",
+    "sinon": "15.2.0",
+    "ts-loader": "9.5.1",
+    "ts-mocha": "10.0.0",
+    "typescript": "5.0.4"
+  },
+  "dependencies": {
+    "@opentelemetry/instrumentation": "^0.203.0"
+  },
```

Comment:
There was a recent change to remove global dev dependencies from individual packages. Try this set:

```
  "devDependencies": {
    "@jsdevtools/coverage-istanbul-loader": "3.0.5",
    "@opentelemetry/api": "^1.3.0",
    "@opentelemetry/api-logs": "^0.208.0",
    "@opentelemetry/sdk-logs": "^0.208.0"
  },
  "dependencies": {
    "@opentelemetry/instrumentation": "^0.208.0"
  },
```